uwblueprint/focus-on-nature
# Waitlist registration entry

## Notion ticket link

[Waitlist Registration: Invite Waitlisted camper to register](https://www.notion.so/uwblueprintexecs/Task-Board-F22-9e03bf9f7c4343f09a8a5c16dc48a12a?p=74da1c05846d4a719c861bca84010cc1&pm=c)



## Implementation description
* changed the model for waitlisted camper so that the contact is split into firstname and last name. This is because the registrant flow needs the primary and secondary contact to have 2 fields for firstname and lastname. 



## Steps to test
1. Go to `http://localhost:3000/register/camp/6397f94e2ff11dcda0ba4960?waitlistedSessionId=6397fa49ef6ca9aaa10cff62&waitlistedCamperId=63ab387d4a2c7a5783a22c94` 
2. You can change the session Id, and mess around with the waitlisted camper on the db like changing expiry date to see that we redirect with error toast 
3. Try testing with other camps too 



## What should reviewers focus on?
* 


## Checklist
